GNUmakefile

```
@@ -123,6 +123,10 @@ test_smoke_raw: test_need_operator_assets ## Run ALL integration smoke tests, on
 test_soak_ocr: test_need_operator_assets ## Run the OCR soak test
 	cd ./integration-tests && go test -v -run ^TestOCRSoak$$ ./soak -count=1 && cd ..
 
+.PHONY: test_soak_forwarder_ocr
+test_soak_forwarder_ocr: ## Run the Forwarder OCR soak test
```

Probably want to follow the same pattern as the other soak tests that Ani recently added which adds the operator assets if necessary.
```suggestion
test_soak_forwarder_ocr: test_need_operator_assets ## Run the Forwarder OCR soak test
```

---

Yep. Thank You for seeing that one ;) added